src/components/Login/Login.tsx

```diff
@@ -1,18 +1,132 @@
-import Button from './Button';
-import Input from './Input';
+import api from '@/Api/api';
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router';
+import { Link } from 'react-router-dom';
+import { HOME_PAGE, SIGNUP_PAGE } from '@/utils/constants';
 import Logo from './Logo';
 
 const Login = () => {
+  const navigate = useNavigate();
+  const [account, setAccount] = useState({
+    email: '',
+    password: '',
+  });
+
+  const onChangeInputValue = (e: any) => {
+    const { name, value } = e.target;
+    setAccount({
+      ...account,
+      [name]: value,
+    });
+  };
+
+  const onClickLoginButton = async () => {
+    const { email, password } = account;
+    try {
+      await api
+        .post(
+          `/login`,
+          {
+            email: email,
+            password: password,
+          },
+          {
+            headers: {
+              'Content-Type': 'application/json',
+            },
+          }
+        )
+        .then((res) => {
+          const token = res.data.token;
+          localStorage.setItem('login-token', token);
+          navigate(HOME_PAGE);
+        });
+    } catch {
+      setError('password', {
+        message: '아이디 또는 비밀번호가 일치하지 않습니다. 다시 입력해 주세요.',
+      });
+    }
+  };
+
+  type formState = {
```

Comment:
```suggestion
  type FormState = {
```

p5;
TS 타입 별칭 네이밍 규칭을 대문자로 시작하고 파스칼 케이스로 많은 사람들이 사용하고 있고, 변수와 타입을 구분할 수 있어서 저도 파스칼 케이스로 타입 별칭을 작성해서 제안드려용~

Reply:
수정했습니다!